configs/STELLARF4V2/config.h

```diff
@@ -0,0 +1,125 @@
+#define USE_MAX7456
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_FLASH_W25M02G
+
+#define BEEPER_PIN          PB2
+#define MOTOR1_PIN          PC9 // TIM8 CH4 (AF3) DMA2 Stream 7 Channel 7
+#define MOTOR2_PIN          PC8 // TIM8 CH3 (AF3) DMA2 Stream 4 Channel 7
+#define MOTOR3_PIN          PC7 // TIM8 CH2 (AF3) DMA2 Stream 3 Channel 7
+#define MOTOR4_PIN          PC6 // TIM8 CH1 (AF3) DMA2 Stream 2 Channel 7
+#define SERVO1_PIN          PB6 // TIM4 CH1 (AF2)
+#define SERVO2_PIN          PB7 // TIM4 CH2 (AF2)
+#define SERVO3_PIN          PB8 // TIM4 CH3 (AF2)
+#define SERVO4_PIN          PB9 // TIM4 CH4 (AF2)
+#define SERVO5_PIN          PA9 // TIM1 CH2 (AF1)
+#define SERVO6_PIN          PA10 // TIM1 CH3 (AF1) or UART1_RX_PIN
+#define SERVO7_PIN          PA8 // TIM1 CH1 (AF1) or UART1_TX_PIN
+#define RX_PPM_PIN          PA3 // TIM5 CH4 (AF2) DMA1 Stream 1 Channel 6
+#define UART2_TX_PIN        PA2
+#define UART3_TX_PIN        PC10
+#define UART4_TX_PIN        PA0
+#define UART5_TX_PIN        PC12
+#define UART2_RX_PIN        PA3
+#define UART3_RX_PIN        PC11
+#define UART4_RX_PIN        PA1
+#define UART5_RX_PIN        PD2
+#define I2C2_SCL_PIN        PB10
+#define I2C2_SDA_PIN        PB11
+#define LED0_PIN            PA15
+#define SPI1_SCK_PIN        PA5
+#define SPI2_SCK_PIN        PB13
+#define SPI3_SCK_PIN        PB3
+#define SPI1_SDI_PIN        PA6
+#define SPI2_SDI_PIN        PC2
+#define SPI3_SDI_PIN        PB4
+#define SPI1_SDO_PIN        PA7
+#define SPI2_SDO_PIN        PB15
+#define SPI3_SDO_PIN        PB5
+#define ADC_VBAT_PIN        PC0
+#define ADC_CURR_PIN        PC1
+#define FLASH_CS_PIN        PC13
+#define PINIO1_PIN          PC14
+#define MAX7456_SPI_CS_PIN  PB12
+#define GYRO_1_EXTI_PIN     PA4
+#define GYRO_1_CS_PIN       PC5
+#define GYRO_1_CLKIN_PIN    PB0 // // TIM3 CH3 (AF2) no DMA
+
+#define TIMER_PIN_MAPPING   TIMER_PIN_MAP( 0,  PC9,  2,  0 ) \
+                            TIMER_PIN_MAP( 1,  PC8,  2,  1 ) \
+                            TIMER_PIN_MAP( 2,  PC7,  2,  1 ) \
+                            TIMER_PIN_MAP( 3,  PC6,  2,  1 ) \
+                            TIMER_PIN_MAP( 4,  PB6,  1, -1 ) \
+                            TIMER_PIN_MAP( 5,  PB7,  1, -1 ) \
+                            TIMER_PIN_MAP( 6,  PB8,  1, -1 ) \
+                            TIMER_PIN_MAP( 7,  PB9,  1, -1 ) \
+                            TIMER_PIN_MAP( 8,  PA9,  1, -1 ) \
+                            TIMER_PIN_MAP( 9,  PA10, 1, -1 ) \
+                            TIMER_PIN_MAP( 10, PA8,  1, -1 ) \
+                            TIMER_PIN_MAP( 11, PA3 , 2,  0 ) \
+                            TIMER_PIN_MAP( 12, PB0,  2, -1 )
+
+#define ADC1_DMA_OPT 0 // DMA2 Stream 0 Channel 0
+
+#define SERIALRX_UART SERIAL_PORT_USART2
+#define SERIALRX_PROVIDER SERIALRX_CRSF
+
+#define MAG_I2C_INSTANCE (I2CDEV_2)
+#define BARO_I2C_INSTANCE (I2CDEV_2)
+#define DASHBOARD_I2C_INSTANCE (I2CDEV_2)
+
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON
```

Comment:
Burst DMA should not be used with F4 MCUs as it is incompatible with bidirectional DShot. 
```suggestion
#define DEFAULT_DSHOT_BURST DSHOT_DMAR_OFF
```